### ISSUE TITLE
Dev version of React & Enable Redux DevTools on Heroku

### ISSUE THREAD

**Issue by oyeanuj** [yury-dymov/redux-oauth#1]
Hi @yury-dymov! Thank you for open-sourcing this, its very helpful.  I just had a couple of questions - 

1. Is it possible to deploy on Heroku with the dev version of redux/react and enable Redux DevTools? This will allow people to better understand how the library is working by inspecting it and playing with it in the devtools?

2. Are the cookies only being used as a storage mechanism? And if yes, would it be possible to configure the library to alternatively use `localStorage`?

Thank you!

**Comment by yury-dymov** [comment]
Hi @oyeanuj, 

1. I like this idea a lot, thank you. I will implement this soon
2. Currently yes. As far as I am concerned each tab has its dedicated localeStorage, which means that authentication might stop working in case user opens application in several different tabs. I am not that sure though, need some time to check this.

**Comment by oyeanuj** [comment]
@yury-dymov Thanks for the quick response!

1. Looking forward to it!
2. `localStorage` persists across tabs. You might be thinking of `sessionStorage`. Here is a reference - http://tutorial.techaltum.com/local-and-session-storage.html

**Comment by yury-dymov** [comment]
@oyeanuj thank you for the url. I am going to check if security-wise localStorage is fine. In this case very likely to implement the feature soon.

**Comment by yury-dymov** [comment]
Hey @oyeanuj

I thought about localeStorage and decided not to implement this feature. My considerations:

 1. Main reason to develop this library was to fully support SSR. So node.js could perform API requests using provided token and provide client whole page right away. Cookies is the only way to implement such behavior as we need somehow to provide token within first browser request. We are not able to run any JS code at that time.
 2. Adding localStorage support will help only client-side users but will also add some pain for SSR users, who for some reasons might misuse this feature
 3. Adding localStorage means increasing size of the library without real payback. Cookies are traditionally used for storing session information so I think that it is a perfect match for token storage.

Basically, I see lots of cons and don't see any pros.

**Comment by yury-dymov** [comment]
Closed via https://github.com/yury-dymov/redux-oauth-demo/commit/96fbc13cc02a7be0c47210541144a325cb39e4b4
and https://github.com/yury-dymov/redux-oauth-client-demo/commit/6101a364351678f6e37f86647da89aeecd250155
